cpu/native/Makefile.dep

```diff
@@ -1,3 +1,12 @@
+ifeq (FreeBSD,$(OS))
+  ifneq ($(DISABLE_LIBSTDCPP),1)
```

Comment:
Unless I'm confused, this should be `ifeq` instead of `ifneq`. I assume that `DISABLE_LIBSTDCPP` is not defined in this context, so it compares `1` with an empty string. So you migh tjust add `DISABLE_LIBSTDCPP ?= 1` for FreeBSD on x86_64

Reply:
Then I will move it from `cpu/native/Makefile.features` to this file.